### PR TITLE
Document IPAM metrics

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: maintenance/monitor/monitor-component-metrics.md [hunk 1]
```diff
@@ -120,6 +126,36 @@ spec:
 EOF
 ```
 
+##### **kube-controllers configuration**
+
+kube-controllers prometheus metrics are **enabled** by default on TCP port 9094. You can adjust the port by modifying the KubeControllersConfiguration resource.
+
+```bash
+calicoctl patch kubecontrollersconfiguration default  --patch '{"spec":{"prometheusMetricsPort": 9095}}'
```
Comment (mgleung) [inline maintenance/monitor/monitor-component-metrics.md:134]: Should this command use `9094` instead of `9095`?

Comment (caseydavenport) [reply maintenance/monitor/monitor-component-metrics.md:134]: I was meaning to show that you can change the deafult - 9094 would be the same as the default.

File: maintenance/monitor/monitor-component-metrics.md [hunk 2]
```diff
@@ -58,6 +61,7 @@ Successfully patched 1 'FelixConfiguration' resource
 ```
 
 ##### **Creating a service to expose Felix metrics**
+
 Promethues uses Kubernetes services to dynamically discover endpoints. Here you will create a service named `felix-metrics-svc` which Prometheus will use to discover all the Felix metrics endpoints.
```
Comment (lwr20) [inline maintenance/monitor/monitor-component-metrics.md:65]:
```suggestion
Prometheus uses Kubernetes services to dynamically discover endpoints. Here you will create a service named `felix-metrics-svc` which Prometheus will use to discover all the Felix metrics endpoints.
```

File: maintenance/monitor/monitor-component-metrics.md [hunk 3]
```diff
@@ -120,6 +126,36 @@ spec:
...
+
+```bash
+calicoctl patch kubecontrollersconfiguration default  --patch '{"spec":{"prometheusMetricsPort": 9095}}'
+```
+
+Setting this value to zero will disable metrics in the kube-controllers pod.
+
+##### **Creating a service to expose kube-controllers metrics**
+
+Promethues uses Kubernetes services to dynamically discover endpoints. Here you will create a service named `kube-controllers-metrics-svc` which Prometheus will use to discover all metrics endpoint.
```
Comment (lwr20) [inline maintenance/monitor/monitor-component-metrics.md:141]:
```suggestion
Prometheus uses Kubernetes services to dynamically discover endpoints. Here you will create a service named `kube-controllers-metrics-svc` which Prometheus will use to discover all metrics endpoint.
```

File: maintenance/monitor/monitor-component-metrics.md [hunk 4]
```diff
@@ -300,16 +350,17 @@ By executing below commands, you will delete all the resource and services creat
...
+kubectl delete service kube-controllers-metrics-svc -n kube-system
 kubectl delete namespace calico-monitoring
 kubectl delete ClusterRole calico-prometheus-user
 kubectl delete clusterrolebinding calico-prometheus-user
 ```
 
 ### Best practices
 
-If you enable {{site.prodname}} metrics to Prometheus, a best practice is to use network policy to limit access to the {{site.prodname}} metrics endpoints. For details, see [Secure {{site.prodname}} Prometheus endpoints]({{ site.baseurl }}/security/comms/secure-metrics).  
+If you enable {{site.prodname}} metrics to Prometheus, a best practice is to use network policy to limit access to the {{site.prodname}} metrics endpoints. For details, see [Secure {{site.prodname}} Prometheus endpoints]({{ site.baseurl }}/security/comms/secure-metrics).
```
Comment (lwr20) [inline maintenance/monitor/monitor-component-metrics.md:361]: https://docs.projectcalico.org/security/comms/secure-metrics only talks about how to secure calico-node and typha.  I think you'll need to add this new component to it.

**Review by mgleung — APPROVED** [review]
LGTM

**Pull request merged**